app/services/guidance_service.rb

```diff
@@ -20,9 +20,22 @@ def orgs
     end
     return @orgs
   end
+
   def any?(org:nil, question:nil)
-    if org.nil? && question.nil?
-      return hashified_annotations? || hashified_guidance_groups?
+    if org.nil?
+      if question.nil?
+        return hashified_annotations? || hashified_guidance_groups?
```

Comment:
This could have just been the `else` to the `if question.present?` below?

Reply:
yes it could.  I'll push an updated version before I leave work today.  Good catch